build/downloadTypos.js

```diff
@@ -0,0 +1,196 @@
+// @ts-check
+/**
+ * OSに合ったtyposのバイナリをダウンロードするスクリプト。
+ */
+const { exec } = require("child_process");
+const { promisify } = require("util");
+const { platform, arch } = require("os");
+const { join, resolve } = require("path");
+const {
+  mkdirSync,
+  existsSync,
+  unlinkSync,
+  createWriteStream,
+  rmSync,
+} = require("fs");
+const fetch = require("node-fetch");
+
+// OS名を定義するオブジェクト
+const OS = {
+  LINUX: "linux",
+  MACOS: "darwin",
+  WINDOWS: "win32",
+};
+// CPUアーキテクチャ名を定義するオブジェクト
+const CPU_ARCHITECTURE = {
+  X86_64: "x86_64",
+  ARM: "aarch64",
+};
+// 全バイナリのパス
+const BINARY_BASE_PATH = resolve(__dirname, "vendored");
+// typosのバイナリのパス
+const TYPOS_BINARY_PATH = resolve(BINARY_BASE_PATH, "typos");
+// 各OSとアーキテクチャに対応するtyposバイナリのダウンロード先URL
```

Comment:
ダウンロード先というと、データの大元ではなく、データの送信先だと思うかも。
まあこうかな？
```suggestion
// 各OSとアーキテクチャに対応するtyposバイナリのダウンロードURL
```